docs/docs/7-integrations/slack-integration.md

```diff
@@ -12,9 +12,99 @@ Testkube bot
 
 ![img.gif](../img/add-testkube-bot-to-workspace.gif)
 
-## **Add the Testkube bot to a Channel**
-
-![img.gif](../img/add-testkube-bot-to-channel.gif)
 ## **Configure Testkube to Use the Slack bot Token**
 
 Populate slackToken in the Helm values file, then install Testkube using Helm install, see [Manual Testkube Helm Charts Installation](../1-installing.md) for more information.
+
+## **Adjust slack config file**
+
+By default the configuration [/charts/testkube-api/slack-config.json](https://github.com/kubeshop/helm-charts/blob/704c71fa3b8f0138f983ea9a2fa598ecbe3868ae/charts/testkube-api/slack-config.json) looks like bellow, it will send notification for all events and all test or testsuite names with any labels.
+If channel is left empty it will send to the first channel that testkube bot is member of.
+
+It is an array of config object, and can use any config combinations
+```json
+[
+    {
+      "ChannelID": "",
+      "selector": {},
+      "testName": [],
+      "testSuiteName": [],
+      "events": [
+        "start-test",
+        "end-test-success",
+        "end-test-failed",
+        "end-test-aborted",
+        "end-test-timeout",
+        "start-testsuite",
+        "end-testsuite-success",
+        "end-testsuite-failed",
+        "end-testsuite-aborted",
+        "end-testsuite-timeout"
+      ]
+    }
+  ]
+```
+e.g.
+
+```json
+[
+    {
+      "ChannelID": "C01234567",
+      "selector": {"label1":"value1"},
+      "testName": ["sanity", "testName2"],
+      "testSuiteName": ["test-suite1", "test-suite2"],
+      "events": [
+        "end-test-failed",
+        "end-test-timeout",
+        "end-testsuite-failed",
+        "end-testsuite-timeout"
+      ]
+    },
+    {
+      "ChannelID": "C07654342",
+      "selector": {"label3":"value4"},
+      "testName": ["integration-test1", "integration-test2"],
+      "testSuiteName": ["integration-test-suite1", "integration-test-suite2"],
+      "events": [
+        "start-test",
+        "end-test-success",
+        "end-test-failed",
+        "end-test-aborted",
+        "end-test-timeout",
+        "start-testsuite",
+        "end-testsuite-success",
+        "end-testsuite-failed",
+        "end-testsuite-aborted",
+        "end-testsuite-timeout"
+      ]
+    },
+]
+```
+
+It will send notifications to channel with id `C01234567` for the test and test suites with labels `label1:value1`, tests "sanity", "testName2" and test suites "test-suite1", "test-suite2" on events "end-test-failed", "end-test-timeout", "end-testsuite-failed", "end-testsuite-timeout" and to channel with id `C07654342` for test with labels `label3:value4`, tests "integration-test1", "integration-test2" and test suites "integration-test-suite1", "integration-test-suite2" on all events.
```

Comment:
```suggestion
This will send notifications to the channel with the id `C01234567` for the test and test suites with labels `label1:value1`, tests "sanity", "testName2" and test suites "test-suite1", "test-suite2" on events "end-test-failed", "end-test-timeout", "end-testsuite-failed", "end-testsuite-timeout" and to the channel with the id `C07654342` for test with labels `label3:value4`, tests "integration-test1", "integration-test2" and test suites "integration-test-suite1", "integration-test-suite2" on all events.
```